jest.config.js

```diff
@@ -1,9 +1,25 @@
 // eslint-disable-next-line no-undef
 module.exports = {
-  preset: 'ts-jest',
+  testPathIgnorePatterns: [
+    "/__tests__/fixtures",
+    "/__tests__/utils",
+  ],
   transform: {
-    '^.+\\.(ts|tsx)?$': 'ts-jest',
+    "^.+\\.tsx?$": "ts-jest",
```

Comment:
Why ?

Reply:
it was jest config from my another project so i just copy and paste and did not pay attention to missing ts in transform